src/plugins/imagePlugin.js

```diff
@@ -237,6 +239,19 @@ export default function createImagePlugin(options = { concept: false }) {
     );
   };
 
+  const generateImageCaptionHtml = embeddedCaption => {
+    const markdown = new Remarkable({ breaks: true, html: true });
+    markdown.inline.ruler.enable(['sub', 'sup']);
+    const caption = embeddedCaption || '';
+    /**
+     * Whitespace must be escaped in order for ^superscript^ and ~subscript~
+     * to render properly. Superfluous whitespace must be escaped in order for
+     * text within *italics* and *bold* to render properly.
+     */
+    const escapedMarkdown = markdown.render(caption.split(' ').join('\\ '));
+    return escapedMarkdown.split('\\').join('');
+  };
```

Comment:
ser dette gjelder andre steder vi tillater sub/sup og, som i ingress i artikler. Kanskje du kunne lagt det til der også eller laget en util som brukte din versjon av denne funksjonen? ⭐

Reply:
om det er just works, ellers er det en egen oppgave. Ingress har en egen preview sak i ed også.

Reply:
Den brukes inntil videre i to forskjellige pakker, og kan muligens også brukes i ingressen. Måtte den isåfall plasseres i frontend-packages, f.eks i util?

Reply:
Trellokortet seier at det skal brukes for lyder, bilder og videoer der vi kan sette egen tekst. Så kanskje lage en util som kan brukes i dei forskjellige pluginsa?

Reply:
Det kan man godt gjøre. Da vil man legge til en ny dependency i frontend-packages, men det går kanskje fint?